src/products/products.service.spec.ts

```
@@ -0,0 +1,31 @@
+        {
+          provide: getRepositoryToken(Product),
+          useClass: Repository,
+        },
+      ],
+    }).compile();
+
+    service = module.get<ProductsService>(ProductsService);
+    repository = module.get<Repository<Product>>(getRepositoryToken(Product));
+  });
+
+  it('should be defined', () => {
```

ESLint

[error] Delete `␍`

(prettier/prettier)